libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -579,6 +579,14 @@ dictionary SwapInfo {
     u32? confirmed_at;
 };
 
+dictionary ListSwapsRequest {
+    sequence<SwapStatus>? status = null;
+    i64? from_timestamp = null;
+    i64? to_timestamp = null;
```

Comment:
Maybe `u64` is better, since epoch time cannot be negative?

Reply:
Makes sense, I copied this from `list_payments`

Reply:
Actually it probably makes more sense to keep the api equivalent between list_payments and this one. So we have the option of making this an i64, or change list_payments to u64. I prefer i64 here, because it doesn't cause breaking changes, what do you think?

Reply:
I see, sounds good.